src/sentry/integrations/pagerduty/notify_action.py

```diff
@@ -37,15 +37,20 @@ def clean(self):
         cleaned_data = super().clean()
 
         integration_id = cleaned_data.get("account")
+        try:
+            integration_id = int(integration_id)
```

Comment:
Shouldn’t this be a validation error?

Reply:
EDIT: I guess this is optional but we should still give a validation error if the param is given and it’s not an int